lib/benchee/profile.ex

```diff
@@ -0,0 +1,98 @@
+defmodule Benchee.Profile do
+  alias Benchee.Output.ProfilePrinter, as: Printer
+  alias Benchee.Suite
+
+  defmodule Benchee.UnknownProfilerError do
+    defexception message: "error"
+  end
+
+  @moduledoc """
+  Profiles each scenario after benchmarking them if the `profile_after` option is either set to:
+    * `true`,
+    * a valid `profiler`,
+    * a tuple of a valid `profiler` and a list of options to pass to it, e.g., `{:fprof, [sort: :own]}`.
+
+  The profiler that will be used is either the one set by the `profiler_after` option or, if set to `true`,
+  the default one (`:cprof`). It accepts however the following profilers:
```

Comment:
so I know I said earlier pick whichever one you want to be the default, reading the short descriptions (thanks!) though I feel like eprof or fprof would be better defaults due to their relation to time vs. function calls (which can be funky).

Hence, I started a quick twitter poll to give us a better lay of the land: https://twitter.com/PragTob/status/1284750740473479169

Reply:
Thanks for making that poll! :)

I'll wait till it finishes before changing all the references to the default profiler. Interesting to note that although José recommends `eprof`, most of the people would like `fprof` (at the time of writing) :thinking:

Reply:
yeah right now it's about even. I think it's reasonable to assume we'll go with `eprof`, if José says fprof is slow that tips the scales into eprofes favor for me. Good metric and fast seems like a good default, of you really wanna dig down `profile_after: :fprof` isn't that much longer than `true` and I think then you know exactly what you want to/getting yourself into :)